### ISSUE TITLE
Nuxt layer start error

### ISSUE THREAD

**Issue by BrutCZE** [nuxt-modules/i18n#1740]
### Environment

RootDir: /app                                                                                                                                                                                                                                                        15:43:11
Nuxt project info:                                                                                                                                                                                                                                                   15:43:11

------------------------------
- Operating System: `Linux`
- Node Version:     `v18.12.1`
- Nuxt Version:     `3.0.0`
- Nitro Version:    `1.0.0`
- Package Manager:  `yarn@3.3.0`
- Builder:          `vite`
- User Config:      `srcDir`, `css`, `build`, `sourcemap`, `typescript`, `vite`, `imports`, `modules`, `i18n`, `nitro`
- Runtime Modules:  `@nuxtjs/i18n@8.0.0-beta.7`, `@pinia/nuxt@0.4.6`, `@vueuse/nuxt@9.6.0`
- Build Modules:    `-`
------------------------------


### Reproduction

I just run `npx nuxi init --template layer nuxt-layer`, add `@nuxtjs/i18n@8.0.0-beta.7` module and run `yarn dev`

### Describe the bug

There will be bug in resolution of `@intlify/vue-router-bridge`.

### Additional context

I found workaround by create symlink `ln -s <projectRoot>/node_modules/ <projectRoot>/.playground/node_modules`.

### Logs

```shell
ERROR  Cannot start nuxt:  Cannot resolve @intlify/vue-router-bridge on yarn! please install it on 'node_modules'                                                                                                                                                   15:36:45

  at tryResolve (node_modules/@nuxtjs/i18n/dist/module.mjs:111:9)
  at async setupAlias (node_modules/@nuxtjs/i18n/dist/module.mjs:233:47)
  at async setup (node_modules/@nuxtjs/i18n/dist/module.mjs:865:5)
  at async normalizedModule (node_modules/@nuxtjs/i18n/node_modules/@nuxt/kit/dist/index.mjs:167:5)
  at async installModule (node_modules/nuxt/node_modules/@nuxt/kit/dist/index.mjs:416:3)
  at async initNuxt (node_modules/nuxt/dist/index.mjs:1825:7)
  at async load (node_modules/nuxi/dist/chunks/dev.mjs:6779:9)
  at async Object.invoke (node_modules/nuxi/dist/chunks/dev.mjs:6840:5)
  at async _main (node_modules/nuxi/dist/cli.mjs:50:20)
```

**Comment by ineshbose** [comment]
Sandbox for reproduction (still need to install `@nuxtjs/i18n`): https://stackblitz.com/github/nuxt/starter/tree/layer

**Comment by kazupon** [comment]
Thank you for your reporting!

We need reproduction github repo or stackbliz.
You can setup with stackbliz that is mentioned by ineshbose

Could you give us it, please?

**Comment by BrutCZE** [comment]
Reproduction repo: https://stackblitz.com/edit/github-5n4ebb?file=nuxt.config.ts

**Comment by kazupon** [comment]
Thanks! I'll fix this issue!

**Comment by kazupon** [comment]
Hi!
I fixed this issue yesterday.
You can try to install edge version `8.0.0-beta.7-27866425.e93ce7b`

Thanks!

**Comment by kazupon** [comment]
That fixed will be included on v8.0.0-beta.8 soon.

**Comment by litingyes** [comment]
> That fixed will be included on v8.0.0-beta.8 soon.

hope it can be faster 🙏🙏🙏
